Sources/Plasma/FeatureLib/pfPatcher/pfPatcher.h

```diff
@@ -84,9 +86,20 @@ class pfPatcher
 
     /** Represents a function that takes (bytesDLed, totalBytes, statsStr) as a progress indicator. */
     typedef std::function<void(uint64_t, uint64_t, const ST::string&)> ProgressTickFunc;
+    
+    /** Represents a function that takes (const plFileName&, hsStream*) on game code discovery.
+     *  You are responsible for closing and deleting the provided stream.
+     */
+    typedef std::function<plFileName(const plFileName&)> FindBundleExeFunc;
```

Comment:
Copy-pasted comment needs updating.

Should probably also move this to the top of the list of `typedef`s, so the order of the `typedef`s remains consistent with the order of the methods.